docs/howto-openshift-crc.md

```diff
@@ -0,0 +1,260 @@
+  samba-operator-controller-manager-7c877459d4-wln54   2/2     Running   0          27s
+```
+
+## Create samba share
+Use the `smbtest.yaml` file below to simple SMB share. Wait for the share pod
+to be in `Running` state (may take some time):
+
+```sh
+  $ export KUBECTL_CMD=oc
+  $ $KUBECTL_CMD create -f smbtest.yaml
+  ...
+  $ $KUBECTL_CMD get pods -n smbtest
+  NAME                      READY   STATUS    RESTARTS   AGE
+  share1-5f7dbd45bc-bljrv   2/2     Running   0          4m23s
+```
+
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: smbtest
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: smb-pv
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 8Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/export"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: smb-pvc
+  namespace: smbtest
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 4Gi
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: users
+  namespace: smbtest
+type: Opaque
+stringData:
+  demousers: |
+    {
+      "samba-container-config": "v0",
+      "users": {
+        "all_entries": [
+          {
+            "name": "user1",
+            "password": "123456"
+          },
+          {
+            "name": "user2",
+            "password": "123456"
+          }
+        ]
+      }
+    }
+---
+apiVersion: samba-operator.samba.org/v1alpha1
+kind: SmbSecurityConfig
+metadata:
+  name: users
+  namespace: smbtest
+spec:
+  mode: user
+  users:
+    secret: users
+    key: demousers
+---
+apiVersion: samba-operator.samba.org/v1alpha1
+kind: SmbCommonConfig
+metadata:
+  name: config
+  namespace: smbtest
+spec:
+  network:
+    publish: cluster
+---
+apiVersion: samba-operator.samba.org/v1alpha1
+kind: SmbShare
+metadata:
+  name: share1
+  namespace: smbtest
+spec:
+  securityConfig: users
+  readOnly: false
+  storage:
+    pvc:
+      name: "smb-pvc"
+...
+```
+
+## Test samba share using smbtoolbox
+Deploy smbtoolbox using the following configuration:
+
+```yaml
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: smbtest
+  name: smbtoolbox
+  annotations:
+    openshift.io/scc: samba
+spec:
+  automountServiceAccountToken: true
+  containers:
+    - name: smbtoolbox
+      image: quay.io/samba.org/samba-toolbox:latest
+      command: ["sleep"]
+      args: ["100000"]
+...
+```
+
+```sh
+  $ export KUBECTL_CMD=oc
+  $ $KUBECTL_CMD create -f smbtoolbox.yaml
+  ...
+  $ $KUBECTL_CMD get pods -n smbtest
+  NAME                      READY   STATUS    RESTARTS   AGE
+  share1-5f7dbd45bc-bljrv   2/2     Running   0          21m
+  smbtoolbox                1/1     Running   0          9m25s
+```
+
+Use the following shell commands and smbclient to test your smbshare:
+
+```sh
+  $ SHARE1_POD=$($KUBECTL_CMD get pods -n smbtest | grep share1 | awk '{print $1}')
```

Comment:
Because github's syntax highligting makes the first option stand out more, I'll note that I do prefer the 2nd option unless we need to reuse SHARE1_POD variable.

Reply:
This is one of those cases where you look at a code snippet and ask yourself: why did the programmer wrote such convoluted code instead of the simple and elegant option? soon enough you realize the *you* are the coder to blame...  

Thanks for this one, will fix.